### ISSUE TITLE
Opt-in open review

### ISSUE THREAD

**Issue by freelanceastro** [openjournals/theoj#176]

**Comment by marcrohloff** [comment]
Is this a duplicate of #177 ?

**Comment by freelanceastro** [comment]
No...this is a feature we'll add in the future that will allow reviewers to decide whether or not they want to sign their reviews (revealing their identity to authors and the other reviewers), and whether or not reviewers and authors collectively decide to make the reviews and responses publicly available. For now, don't worry about it, and just enforce the privacy restrictions as outlined in #177.
